src/ty.md

```diff
@@ -78,7 +78,7 @@ is in fact a simple type alias for a reference with `'tcx` lifetime:
 pub type Ty<'tcx> = &'tcx TyS<'tcx>;
 ```
 
-[the HIR]: ../hir/README.md
+[the HIR]: https://github.com/rust-lang/rust/blob/master/src/librustc/hir/README.md
```

Comment:
I think we *do* have a HIR chapter, though... can we redirect to [the HIR page instead](https://rust-lang-nursery.github.io/rustc-guide/hir-lowering.html)?

Reply:
Hmm... that link is showing up as a 404 for me. Has the PR for it landed yet?

Reply:
Ah, looks like it's [hir.html](https://rust-lang-nursery.github.io/rustc-guide/hir.html), not `hir-lowering.html`.